feat(query-tee): Support query-tee comparisons for multiple secondary backends (#13022)
grafana/mimir: CHANGELOG.md

* [ENHANCEMENT] Improve the description of how to limit the number of buckets in native histograms. #12797
* [BUGFIX] Add a missing attribute to the list of default promoted OTel resource attributes in the docs: deployment.environment. #12181

### Query-tee

> * [CHANGE] If multiple secondary backends are configured and comparisons are enabled, query-tee will now report comparison results of the preferred backend against each of the secondaries #13022

Reviewer: ```suggestion
* [CHANGE] If you configure multiple secondary backends and enable comparisons, query-tee reports comparison results of the preferred backend against each of the secondaries. #13022
```
